src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -521,10 +525,14 @@ def fetch(self, rid):
                 transitions.remove(status)
             visible_attrs = get_modifiable_properties(status)
             filterout_attrs = get_protected_properties()
-
             # extend filterout list with "RequestStatus" since it is passed separately
             filterout_attrs.append("RequestStatus")
 
+            args = self.assignArgs
+            for key, defin in self.assignArgs.iteritems():
```

Comment:
I think Eric had a good argument to call it `iteritems()` instead of `items`, keeping in mind the python 3 migration. But I'll update it